components/more_direct_channels/list_item/list_item.test.tsx

```diff
@@ -6,6 +6,8 @@ import React from 'react';
 
 import {General} from 'mattermost-redux/constants';
 
+import {OptionValue} from '../types';
+
 import ListItem from './list_item';
 
 describe('ListItem', () => {
```

Comment:
Thanks @cannalee90 can you please add a type for the props?

Reply:
no problem. I add the props and apply it

Reply:
@nevyangelova Hi. Would you resolve this issue?